src/Blocks/components/post-card/post-card-style.scss

```diff
@@ -0,0 +1,42 @@
+.post-card {
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+
+	&__image {
+		padding-bottom: 20px;
+		height: 100%;
+		width: 100%;
+		object-fit: cover;
+	}
+
+	&__image-wrap {
+		height: 60%;
```

Comment:
this is highly specific what does this do?

Reply:
That was something I tried before I realized there was a bug with columns in Safari which your PR later fixed.

Looks like I forgot to include one of the commits, that'll be solved now.